src/libPMacc/examples/gameOfLife2D/include/Simulation.hpp

```diff
@@ -113,27 +113,42 @@ class Simulation
          * scheme using Offsets from global LEFT, TOP, FRONT                  */
         const SubGrid<DIM2>& subGrid = Environment<DIM2>::get().SubGrid();
 
-        /* Recall that in types.hpp the following is defined:                 *
+        /* The following sets up the local layout which consists of the actual
+         * grid cells and some surrounding cells, called guards.
+         * Visualization (1D, 2 GPUs, only 1 border shown, actually some in all directions):
+         * _______GPU 0________       _______GPU 1________
+         * | 0 | 1 | 2 | 3 | 4 |      | 3 | 4 | 5 | 6 | 7 |  <-- Global (super)cell idx
+         * |___|___|___|___|___|      |___|___|___|___|___|
+         * |___Core____|Bor|Gua|      |Gua|Bor|___Core____|
+         * |___________|der|rd_|      |rd_|der|___________|
+         * |___Workspace___|              |___Workspace___|
```

Comment:
nice ascii gfx, a small remark:
we don't use the term "workspace" in the rest of the code, even if it seems reasonable at this point. it is also not very clear because depending on the individual algorithm, it's hard to tell what "is work" and what not (the guard might participate here and this makes things less clear).

I would rather put a sentence that guards are `cloned cells from the neighbor (sometimes also referred to as "ghost" or "halo cells")`. One could rename "workspace" with "real cells" or "original cells".
```txt
* ASCII visualization: example taken for 1D,
*   distributed over 2 GPUs, only 1 border between those two
*   GPUs shown assuming non-periodic boundary conditions.
*   In a N-GPU or periodic example, border cells are marked and
*   guard cells are placed in each direction.
* _______GPU 0________ _______GPU 1________
* | 0 | 1 | 2 | 3 | 4 | | 3 | 4 | 5 | 6 | 7 | <-- Global (super)cell idx
* |___|___|___|___|___| |___|___|___|___|___|
* |___Core____|Bor|Gua| |Gua|Bor|___Core____|
* |___________|der|rd_| |rd_|der|___________|
* |___org. cells__|***| |***|___org. cells__|
*
*  |***| Cloned cells from the border cells of the neighbor GPU
*        (sometimes also referred to as "ghost" or "halo cells").
* [...]
*/
```